AllReadyApp/Web-App/AllReady/Features/Notifications/NotifyAdminForTaskSignupStatusChange.cs

```diff
@@ -36,12 +36,9 @@ public void Handle(TaskSignupStatusChanged notification)
             var activity = task.Activity;
             var campaign = activity.Campaign;
 
-            //Prefer activity organizer email; fallback to campaign organizer email
-            var adminEmail = taskSignup.Task.Activity.Organizer?.Email;
-            if (string.IsNullOrWhiteSpace(adminEmail))
-            {
-                adminEmail = taskSignup.Task.Activity.Campaign.Organizer?.Email;
-            }
+            var campaignContact = campaign.CampaignContacts.SingleOrDefault(tc => tc.ContactType == (int)ContactTypes.Primary);

```

Comment:
Handle the possibility that `CampaignContacts` could be null. `var campaignContact = campaign.CampaignContacts?.SingleOrDefault(...)`